src/marine/ndbc_hfradar2ioda.py

```diff
@@ -22,15 +22,14 @@
 from collections import defaultdict, OrderedDict
 from orddicts import DefaultOrderedDict
 
-vName = ["sea_water_meridional_current",
-         "sea_water_zonal_current",
-         "DOPX",
-         "DOPY"]
+vName = ["waterMeridionalCurrentSpeed",
```

Comment:
I will check the WMO element table, but perhaps we can use ```waterMeridionalVelocity``` and ```waterZonalVelocity```

Reply:
@gthompsnJCSDA those sound fine. I would prefer velocity over speed. Should we have a `...SurfaceVelocity` as well for the same reasons we have the other surface variables? what does the atmosphere do in this case? The case of HF radar here is similar to SST/SSS, whereby the "surface" measurement is some ill defined almost-at-the-surface depth.

Reply:
Now I used vComponentOfCurrent and uComponentOfCurrent right now. Please check WMO table and let me know your suggestions.

Reply:
@gthompsnJCSDA, @travissluka, in WMO, it has only speed of current and direction of current. But I also found:
SEA/WATER CURRENT SPEED AT A LEVEL
SEA/WATER CURRENT DIRECTION AT A LEVEL
SEA SURFACE CURRENT SPEED
There is no other definitions. Only reference is from NCEP GRIB table definition for x-direction and y direction current definitions.